src/main/java/hu/progmasters/probac/Morse.java

```diff
@@ -0,0 +1,106 @@
+package hu.progmasters.probac;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Converts string into Morse Code
+ */
+public class Morse {
+    /**
+     * Starts main code
+     * @param args String array standard start parameter
+     */
+    public static void main(String[] args) {
+        Morse morse = new Morse();
+        morse.printMorseCode("Wozu Worte drucken, es gibt doch Schreiber");
+    }
+
+    /**
+     * Returns index of letter in alphabet.
+     * @param letter String whose index is to be found in alphabet.
+     * @return index of letter
+     */
+    int getPosInAlphabet(String letter) {
+        if (letter == null) {
+            letter = "";
+        }
+        letter = letter.toLowerCase();
+        String alphabet = "abcdefghijklmnopqrstuvwxyz1234567890";
+        return alphabet.indexOf(letter);
+    }
+
+    /**
+     * prints String converted into Morse code
+     * @param input String that is to be converted
+     */
+    void printMorseCode(String input) {
+        input = input.toLowerCase();
+        input = input.replaceAll("[^a-z0-9]", ""); // removes everything but letters and digits
+
+        String[] morsecode = {
+                ".-",   // A
+                "-...", // B
+                "-.-.", // C
+                "-..",  // D
+                ".",    // E
+                "..-.", // F
+                "--.",  // G
+                "....", // H
+                "..",   // I
+                ".---", // J
+                "-.-",  // K
+                ".-..", // L
+                "--",   // M
+                "-.",   // N
+                "---",  // O
+                ".--.", // P
+                "--.-", // Q
+                ".-.",  // R
+                "...",  // S
+                "-",    // T
+                "..-",  // U
+                "...-", // V
+                ".--",  // W
+                "-..-", // X
+                "-.--", // Y
+                "--..", // Z
+                ".----", // 1
+                "..---", // 2
+                "...--", // 3
+                "....-", // 4
+                ".....", // 5
+                "-....", // 6
+                "--...", // 7
+                "---..", // 8
+                "----.", // 9
+                "-----"  // 0
+        };
+
+
+        List<String> results = ConvertText2Morse(input, morsecode);
+        //System.out.println(); // new line at the end
+        String noBrackets = Arrays.toString(results.toArray()).replace("[", "").replace("]", "");
+        System.out.println(noBrackets);
+    }
+
+    public List<String> ConvertText2Morse(String input, String[] morsecode) {
```

Comment:
Pls. write a javadoc for Method ConvertText2Morse(...)

Reply:
Javadoc updated.